packages/features/ee/workflows/lib/compareReminderBodyToTemplate.ts

```diff
@@ -0,0 +1,16 @@
+const compareReminderBodyToTemplate = ({
+  reminderBody,
+  template,
+}: {
+  reminderBody: string;
+  template: string;
+}) => {
+  const stripHTML = (html: string) => html.replace(/<[^>]+>/g, "").replace(/&amp;/g, "&");
```

Comment:
`&` is encoded / not encoded between the editor and the template. Decided to keep both templates the same by adding `&`